### PR TITLE
PLF-7090 : add Tomcat 8.5 upgrade in release notes

### PR DESCRIPTION
**Pull request by thomasdelhomenie**

### CODE REVIEW

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt [hunk 1]
```diff
@@ -136,7 +136,20 @@ However, if you still insist on skipping versions, we strongly advise to read al
...
 
-== Upgrading from Platform 4.4.0 to ${org.exoplatform.platform.version} ==
+== Upgrading from Platform 4.4.1 to ${org.exoplatform.platform.version} ==
+
+=== Description of the plugins used in this upgrade ===
+
+=== Major Components Upgrades ===
+
+* Tomcat 8.5
+Tomcat has been upgraded from 7.0 version to 8.5. The main changes related to this upgrade are :
```
Comment (mgreau) [inline plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: remove whitespace before the colon

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt [hunk 2]
```diff
@@ -136,7 +136,20 @@ However, if you still insist on skipping versions, we strongly advise to read al
...
+** conf/Catalina/localhost/context.xml.default has been removed, and all customizations related to Context have been gathered in conf/context.xml.
+** the property org.apache.catalina.startup.ContextConfig.jarsToSkip has been removed from conf/catalina.properties since it is not supported anymore and has been replaced by the JarScanning tag in conf/context.xml. The scanManifest has also been disabled (feature introduced in Tomcat 8.0) to avoid dependencies issues at startup.
+** war files are now exploded by default. This is a change introduced by Tomcat 8.0 (https://tomcat.apache.org/migration-8.html#Tomcat_8.0.x_noteable_changes). The variable SET EXO_TOMCAT_UNPACK_WARS can be set false (defaults to true) in setenv-customize.sh|bat to not explode wars but this can lead to slower startup and performance issues, as mentioned in the link above.
+** Tomcat 8.5 comes with Servlet 3.1 (was 3.0 in Tomcat 7.0), JSP 2.3 (was 2.2 in Tomcat 7.0), EL 3.0 (was 2.2 in Tomcat 7.0)
```
Comment (mgreau) [inline plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: > can be set false (defaults to true)

can be set **to** false (true by default)

**Review by mgreau — CHANGES_REQUESTED** [review]

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt (hunk 1 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: Done

File: plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt (hunk 2 above)
Comment (thomasdelhomenie) [reply plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt]: Done

**Review by mgreau — APPROVED** [review]

**Pull request merged**